package.json

```diff
@@ -5,8 +5,10 @@
   "description": "A bundle of components which Gaia needs.",
   "author": "The Gaia Team <dev-gaia@lists.mozilla.org>",
   "devDependencies": {
-    "async": "0.2.9",
```

Comment:
The npm tool has reordered the package.json when I installed [app-root-path](https://www.npmjs.com/package/app-root-path).